src/utils/token/index.test.js

```diff
@@ -16,7 +22,42 @@ describe('testing function related to the token', () => {
     await setRefreshToken({id: '1234'});
     expect(await getRefreshToken()).toEqual('{"id":"1234"}');
   });
+
+  it('setToken should run correctly', async () => {
+    const spyStorage = jest.spyOn(AsyncStorage, 'setItem');
+    await setToken('123456');
+    expect(spyStorage).toHaveBeenCalled();
+  });
+  it('setAnonymousToken should run correctly', async () => {
+    const spyStorage = jest.spyOn(AsyncStorage, 'setItem');
+    await setAnonymousToken('123456');
+    expect(spyStorage).toHaveBeenCalled();
+  });
+  it('getToken should run correctly', async () => {
+    const spyStorage = jest.spyOn(AsyncStorage, 'getItem').mockResolvedValue('6I7SIFD7BPSKZGK0Y6');
+    const store = await getToken();
+    expect(spyStorage).toHaveBeenCalled();
+    expect(store).toEqual('6I7SIFD7BPSKZGK0Y6');
+  });
+  it('setUserId should run correctly', async () => {
+    const spyStorage = jest.spyOn(AsyncStorage, 'setItem');
+    await setUserId('123456');
+    expect(spyStorage).toHaveBeenCalled();
+  });
+  it('getUserIdshould run correctly', async () => {
+    const spyStorage = jest.spyOn(AsyncStorage, 'getItem').mockResolvedValue('6I7SIFD7BPSKZGK0Y6');
+    const store = await getUserId();
+    expect(spyStorage).toHaveBeenCalled();
+    expect(store).toEqual('6I7SIFD7BPSKZGK0Y6');
+  });
+
+  it('removeLocalStorage run correctly', async () => {
+    const spyStorage = jest.spyOn(AsyncStorage, 'removeItem');
+    await removeLocalStorege();
+    expect(spyStorage).toHaveBeenCalled();
+  });
 });
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a typo in the function name `removeLocalStorege` on line 56. It should be `removeLocalStorage`.
```diff
-    await removeLocalStorege();
+    await removeLocalStorage();
```
Also, it would be better to add assertions for the values stored and retrieved from AsyncStorage in your tests. This will ensure that the correct values are being set and returned.
For example, in the test for `setToken` (lines 26-30), you could add an assertion to check if the correct token was stored:

```javascript
it('setToken should run correctly', async () => {
  const spyStorage = jest.spyOn(AsyncStorage, 'setItem');
  await setToken('123456');
  expect(spyStorage).toHaveBeenCalledWith('token', '123456');
});
```

You can apply similar changes to other tests as well.


<!-- This is an auto-generated comment by OSS CodeRabbit -->